llabake/Hello-Books
# Ch paginate pages 157932781

#### What does this PR do?
This PR implements pagination on the front end with pagination data from the backend 

#### Description of Task to be completed?
  *  implement front end pagination on favorite book page
   * add front end pagination to user profile page
   * add front end pagination to admin dashboard page
   * add action type to hold response count
   * manage state of response count in their respective reducer
   * add maximum page limit to webpack common
   * render new list and page count of reduction of pagination data content
   * get current state of admin reducer and reduce the count for admindashboard pagination meta
   * fix proptype validation error on side nav component
#### How should this be manually tested?
* clone the repo
* cd into the project directory
* run npm install to install all the dependency
* run npm run start-dev to start the back-end server
* run npm run dev to start the front-end server
* navigate to a page that displays an array
* pagination indicator should be shown
#### What are the relevant pivotal tracker stories?
[finishes #157932781]

#### Any background context you want to add?
* The application was fetching all results from the back end previously and then displaying the result in sets

#### Screenshots

N/A
